jekyll/_cci2/configuration-reference.md

```diff
@@ -61,18 +61,21 @@ executors | N | Map | A map of strings to executor definitions. See the [Executo
 commands | N | Map | A map of command names to command definitions. See the [Commands]({{ site.baseurl }}/2.0/configuration-reference/#commands-requires-version-21) section below.
 {: class="table table-striped"}
 
-The following example calls an Orb named `hello-build` that exists in the certified `circleci` namespace.
+The following example calls an Orb named `node` that exists in the certified `circleci` namespace.
```

Comment:
```suggestion
The following example calls an orb named `node` that exists in the certified `circleci` namespace.
```

Reply:
This was already there so not your work but FYI we don't generally capitalise feature names :-)